admin/main.js

```diff
@@ -0,0 +1,3 @@
+function main() {
+    
+}
```

Comment:
Missing new line :)

Reply:
Ok, please, can you explain me why is this important? I would like to learn about this

Reply:
See comment above

Reply:
Thanks, I didn't realize that comments appear above